packages/mcp-client/src/Agent.ts

```diff
@@ -44,7 +45,7 @@ const askQuestionTool: ChatCompletionInputTool = {
 const exitLoopTools = [taskCompletionTool, askQuestionTool];
 
 export class Agent extends McpClient {
-	private readonly servers: StdioServerParameters[];
+	private readonly servers: (ServerConfig|StdioServerParameters)[];
```

Comment:
you'll want to run the linter

Reply:
if you're using VS Code, it should pick up the formatting config automatically, otherwise:

pnpm lint and pnpm format